cli/setup/setup.go

```diff
@@ -0,0 +1,88 @@
+package setup
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/rs/zerolog/log"
+	"github.com/urfave/cli/v2"
+
+	"go.woodpecker-ci.org/woodpecker/v2/cli/internal/config"
+	"go.woodpecker-ci.org/woodpecker/v2/cli/setup/ui"
+)
+
+// Command exports the setup command.
+var Command = &cli.Command{
+	Name:  "setup",
+	Usage: "setup the woodpecker-cli for the first time",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:  "server-url",
+			Usage: "The URL of the woodpecker server",
+		},
+		&cli.StringFlag{
+			Name:  "token",
+			Usage: "The token to authenticate with the woodpecker server",
+		},
+	},
+	Action: setup,
+}
+
+func setup(c *cli.Context) error {
+	_config, err := config.Get(c, c.String("config"))
+	if err != nil {
+		return err
+	} else if _config != nil {
+		setupAgain, err := ui.Confirm("The woodpecker-cli was already configured. Do you want to configure it again?")
+		if err != nil {
+			return err
+		}
+
+		if !setupAgain {
+			log.Info().Msg("Configuration skipped")
+			return nil
+		}
+	}
+
+	serverURL := c.String("server-url")
+
+	if serverURL == "" {
+		serverURL, err = ui.Ask("Enter the URL of the woodpecker server", "https://woodpecker-ci.org", true)
```

Comment:
Does it make sense to use a placeholder that is an existing URL but not a Woodpecker CI server? I would suggest using either `"https://ci.woodpecker-ci.org"` or `"https://ci.example.com"`.

Reply:
```suggestion
		serverURL, err = ui.Ask("Enter the URL of the woodpecker server", "https://ci.woodpecker-ci.org", true)
```